libraries/difxmessage/utils/statemon

```diff
@@ -19,6 +19,7 @@
 #   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
 #**************************************************************************
 
+
```

Comment:
One blank line was inadvertently added here (just proving that I was paying attention!)